cfgov/unprocessed/apps/regulations3k/js/analytics.js

```diff
@@ -25,7 +24,7 @@ const sendEvent = (action, label, category) => {
  * @returns {HTMLElement|null} The expandable or null if it's not an expandable
  */
 const getExpandable = (event) => {
-  const el = closest(event.target, '.o-expandable_header') || event.target;
+  const el = event.target.closest('.o-expandable_header') || event.target;
```

Comment:
TRIVIAL: This could be slightly optimized to:

```js
const target = event.target;
const el = target.closest('.o-expandable_header') || target;
```

As is, if closest returns `null`, `event.target` will be fetched twice.

Reply:
I don't think this is worth optimizing for speed as it's not in a hot path. In general, if performance of this code were critical, there would be structural changes we could make that would speed it up, but ultimately, an additional object access will not be measurable in response to a click event. Either way, we'd want to do that via performance profiling.

As well, playing devil's advocate, there is a cost to allocating the `target` variable, which may outweigh the additional access.

Reply:
Yea either is fine. From a readability perspective I have a slight preference for the shorter lines.